pkg/loop/server.go

```diff
@@ -85,7 +85,7 @@ func (s *Server) start() error {
 		return fmt.Errorf("error starting prometheus server: %w", err)
 	}
 
-	s.checker = services.NewChecker()
+	s.checker = services.NewChecker("", "")
```

Comment:
Where would we expect `NewChecker` to be called with non-empty string values for `ver` and `sha`?